app/helpers/collections_helper.rb

```diff
@@ -1,11 +1,11 @@
 module CollectionsHelper
   DOWNLOAD_XML_OPTIONS = [
-    { format: 'native',   title: 'NATIVE' },
+    #{ format: 'native',   title: 'NATIVE' },
     { format: 'atom',     title: 'ATOM' },
     { format: 'dif10',    title: 'DIF 10' },
     { format: 'echo10',   title: 'ECHO 10' },
-    { format: 'iso',      title: 'ISO 19115 (MENDS)' },
-    { format: 'iso19115', title: 'ISO 19115 (SMAP)' }
+    { format: 'iso19115',      title: 'ISO 19115 (MENDS)' },
```

Comment:
I heard a comment was going to be added here explaining that we redefined iso19115